coordinator/tests/services/validation/bundle-validator.spec.ts

```diff
@@ -446,6 +449,16 @@ describe("MedicationRequest consistency checks", () => {
     expect(validationErrors).toHaveLength(1)
   })
 
+  test("Should throw error when GMP-number is only number in Practitioner.identifier", () => {
+    const testReference: fhir.Identifier = createIdentifier("https://fhir.hl7.org.uk/Id/gmp-number", "G1234567")
+    if (isReference(practitionerRoles[0].practitioner)) {
+      const practitioner = resolveReference(bundle, practitionerRoles[0].practitioner)
+      practitioner.identifier = [testReference]
+    }
+
+    const validationErrors = validator.verifyPrescriptionBundle(bundle)
+    expect(validationErrors).toHaveLength(1)
```

Comment:
Hi @TomSmithCoding,  We could check the error details in the expect statement to ensure we are throwing the correct error for this scenario.

Reply:
Sounds good! Good to be specific, will get this added.